cmd/agent/app/check.go

```diff
@@ -242,6 +242,11 @@ var checkCmd = &cobra.Command{
 		var instancesData []interface{}
 
 		for _, c := range cs {
+			for _, conf := range allConfigs {
+				if check.IsJMXConfig(conf.Name, conf.InitConfig) {
+					return fmt.Errorf("using the jmx option with the check command directly is deprecated, please use the jmx command instead")
```

Comment:
same here, I would just say that it's not supported instead of mentioning a deprecation